com.unity.multiplayer.mlapi/Prototyping/NetworkAnimatorVar.cs

```diff
@@ -0,0 +1,490 @@
+using System.Collections.Generic;
+using System.Linq;
+using MLAPI.NetworkVariable;
+using MLAPI.Serialization;
+using MLAPI.Transports;
+using UnityEngine;
+
+namespace MLAPI.Prototyping
+{
+    /// <summary>
+    /// A prototype component for syncing animations
+    /// </summary>
+    [AddComponentMenu("MLAPI/NetworkAnimatorVar")]
+    public class NetworkAnimatorVar : NetworkBehaviour
+    {
+        /// <summary>
+        /// Server authority only allows the server to update this animator
+        /// Client authority only allows the client owner to update this animator
+        /// </summary>
+        public enum Authority
+        {
+            Server = 0,
+            Owner,
+        }
+
+        public Animator Animator => m_Animator;
+        
+        /// <summary>
+        /// TODO this will need refactoring
+        /// Specifies who can update this animator
+        /// </summary>
+        [Tooltip("Defines who can update this transform.")]
+        public Authority AnimatorAuthority = Authority.Owner;
+        
+        [SerializeField] private Animator m_Animator;
+
+        /// <summary>
+        /// The base amount of sends per seconds to use when range is disabled
+        /// </summary>
+        [Range(0, 120), Tooltip("The base amount of sends per seconds to use when range is disabled")]
+        public float FixedSendsPerSecond = 30f;
+
+        /// <summary>
+        /// The channel to send the data on
+        /// </summary>
+        [Tooltip("The channel to send the data on.")]
+        public NetworkChannel Channel = NetworkChannel.NetworkVariable;
+        
+        private NetworkVariable<AnimatorSnapshot> m_AnimatorState = new NetworkVariable<AnimatorSnapshot>();
```

Comment:
I think this will not work when authority is set to owner because the default write permission for NetworkVariable is server only.

Reply:
Ah never mind I just saw it gets initialized again in NetworkStart correctly. Maybe don't initialize it here then.